src/setup/cortx_deploy

```diff
@@ -145,11 +145,11 @@ cluster_id=$(conf_get $(yaml_config $CLUSTER_INFO) "cluster>id")
 log $INFO "Starting provisioning on node $node_id which is part of cluster $cluster_id."
 
 # Check current image version
-installed_version=$(sed -e 's/[][]//g' -e 's/"//g' <<< $(conf_get $CORTX_CONFSTORE "cortx>common>release>version"))
+installed_version=$(conf_get $CORTX_CONFSTORE "node>$node_id>provisioning>version")
```

Comment:
Do we handle missing key as well?

Reply:
Yes if key is missing it gives null value and that condition is already handled in code. L-150.